base/shell/explorer/traywnd.cpp

```diff
@@ -3045,10 +3045,8 @@ class CTrayWindow :
 
     LRESULT OnNcLButtonUp(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
     {
-        if (CheckShowDesktopButtonClick(lParam, bHandled))
-            return 0;
-
-        return 0;
+        CheckShowDesktopButtonClick(lParam, bHandled);
+        return FALSE;
```

Comment:
Why FALSE?
The return is an LRESULT, not a BOOLEAN

Reply:
I think this is following existing code in this file. There are other functions which were coded the same way.

Reply:
@learn-more yes, I did orienedt at all the surrounding functions here that returned LRESULT as TRUE/FALSE.
Do you want me to change them all? Or can I push it like that? I don't have any preference in that regard.

Reply:
TBH this PR already cost more time than it will ever gain.